docs/guides/distribution/app-stores.md

```diff
@@ -0,0 +1,86 @@
+# Publishing Tauri Applications to App Stores
+
+In this guide, we will cover the process of publishing Tauri applications on various app stores, including Snapstore (Linux), Flathub (Linux), Chocolatey (Windows), and AppImage (Linux). Please note that the macOS platform is not covered due to the unavailability of a testing device.
+
+## Snapstore
+
+**Note**: This process requires a Linux operating system, and WSL may not be suitable depending on the application type.
+
+### Method 1: Utilizing an External npm Library
+
+1. Begin by creating a `.snap` package for your application using the [tauri-snap-packager](https://www.npmjs.com/package/tauri-snap-packager) npm package.
+2. Install the package: `npm install --save-dev tauri-snap-packager`
+3. Update your `package.json` file to include the tauri-snap command.
+   ```json
+   "scripts": {
+       "dev": "next dev -p 1420",
+       "build": "next build && next export -o dist",
+       "tauri": "tauri",
+       "lint": "next lint",
+       "tauri-snap": "tauri-snap-packager"
+   },
+   ```
+4. Run the `tauri-snap` command to generate a `.snap` file in your target folder, following any on-screen instructions.
+5. In case of errors, rerun the command with the `--trace-warnings` flag for detailed feedback.
+6. If issues persist, consider Method 2 as an alternative.
+7. Upon successful creation of the `.snap` file, refer to the [Snapcraft guide](https://snapcraft.io/first-snap/c/linux/upload) for uploading and publishing it to the store.
+8. Customize logos, banners, descriptions, etc., on the [Snapcraft website](https://snapcraft.io/). Refer to the [release management guide](https://snapcraft.io/docs/release-management) for additional details.
+
+### Method 2: Starting from Scratch
+
+If the automatic tool is unsuitable, manually create a `snapcraft.yaml` file.
+- Install `snapcraft` and `snapd` for your operating system. Refer to the [Snapcraft guide](https://snapcraft.io/snapcraft) for instructions.
+- Create a `snapcraft.yaml` file and configure it according to your application's specifications.
+- Execute the `snapcraft` command to generate the `.snap` file.
+- Follow the [Snapcraft guide](https://snapcraft.io/first-snap/c/linux/upload) for uploading and publishing on the store.
+
+## Flathub
+
+**Note**: Flathub support is currently unavailable, as discussed in [this GitHub thread](https://github.com/tauri-apps/tauri/discussions/4426).
```

Comment:
That's not really true, it's basically like Method 2 of the snap support if you're re-using the .deb bundle. There are even a few tauri apps on flathub already. That said, it only really makes sense for tauri v2 apps since v1 requires a webkitgtk version that's not part of the newer runtimes anymore.

Either way, we can add that in another PR :)

Reply:
I can't wait for v2 to release, I have only heard good things about it so far